### PR TITLE
BMI Calculator UI is 100% done.  Functionality is 80% completed.

### PR DESCRIPTION
**Pull request by DerekKensinger**
### What was the problem or new feature?

Completed the UI for the BMI calculator.  It is working and I have set up functions to validate the user input so that it is working as intended.  I have to add the "Suggestions" page based on their BMI.

### How does this PR fix the problem or implement the feature?

I have figured out how to use card views to make the app flow smoother in the Physical section

### Checklist

- [X] Coding style [Trailing whitespaces in files & no unneeded whitespaces] :sparkles:
- I might have forgotten some trailing whitespace on one of my files.
- [X ] Are there any .idea/ files or .gradle/ files in this PR? :notebook: No

### What issue(s) does this PR close?

Closes the BMI Calculator

### Additional Comments (if any):

Hope you all are well and I will see you on Friday for the build review if I am not flying home then.  My flight is still up in the air with the hurricane and am currently stranded in New Orleans.

### CODE REVIEW

File: app/src/main/java/com/example/life_is_well/BMICalc.kt [hunk 1]
```diff
@@ -1,11 +1,83 @@
...
+        val heightText = findViewById<EditText>(R.id.etHeight)
+        val calcButton = findViewById<Button>(R.id.btnCalculate)
+
+        calcButton.setOnClickListener {
+            val weight = weightText.text.toString()
+            val height = heightText.text.toString()
+            if(validateInput(weight, height)){
+                // BMI equation for Standard Measurement System (Pounds & Inches)
+                val bmi = (weight.toFloat()/((height.toFloat())*(height.toFloat()))) * 703
+                // Setting the result to two decimal places, Stackoverflow came in clutch here
```
Comment (jacobburgo) [inline app/src/main/java/com/example/life_is_well/BMICalc.kt:28]: Don't be afraid to put the link of the Stackoverflow right in the comment. Could prove to be helpful to others and can't do any harm! 👍🏼

File: app/src/main/java/com/example/life_is_well/BMICalc.kt [hunk 2]
```diff
@@ -1,11 +1,83 @@
...
+import androidx.core.content.ContextCompat
 
 class BMICalc : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_bmicalc)
+
+        val weightText = findViewById<EditText>(R.id.etWeight)
+        val heightText = findViewById<EditText>(R.id.etHeight)
+        val calcButton = findViewById<Button>(R.id.btnCalculate)
```
Comment (jacobburgo) [inline app/src/main/java/com/example/life_is_well/BMICalc.kt:20]: Nice! This is much better than importing entire xml files into the .kt files!

File: app/src/main/res/layout/activity_bmicalc.xml [hunk 3]
```diff
@@ -4,25 +4,25 @@
...
         android:id="@+id/layoutHeaderAccountMain"
         android:layout_width="432dp"
         android:layout_height="wrap_content"
-        android:background="@drawable/home_page_header_background"
+        android:background="@drawable/home_page_health_btn_background"
         android:padding="10dp"
         app:layout_constraintTop_toTopOf="parent"
-        tools:layout_editor_absoluteX="-11dp"
-        tools:ignore="MissingConstraints">
+        tools:ignore="MissingConstraints"
```
Comment (jacobburgo) [inline app/src/main/res/layout/activity_bmicalc.xml:17]: What is the reason for this? You've been in the code more than I so I'll defer to you, but everything I've watched emphasized the importance of constraints, unless this is not doing what it sounds like.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -92,4 +92,11 @@
     <string name="LovingLife">What I am Loving about Life right now:</string>
     <string name="social_media">Social Media</string>
     <string name="users_profile_picture">Users Profile Picture</string>
+    <string name="weight">Weight:</string>
+    <string name="_0">0</string>
+    <string name="pounds_lbs">Pounds (lbs)</string>
+    <string name="height">Height:</string>
+    <string name="inches">Inches</string>
+    <string name="calculate">Calculate</string>
+    <string name="normal_range">(Normal Range is 18.5 - 24.9)</string>
```
Comment (jacobburgo) [inline app/src/main/res/values/strings.xml:101]: Something to note moving forward is that if we are going to implement "Themes" in the settings (night/light/other), we want to keep the `night/strings.xml` file up-to-date with this file. Not sure about on your machine, but mine spits out warnings/errors when the two files don't match.

**Review by jacobburgo — COMMENTED** [review]
Just a couple comments/questions before approving.

File: app/src/main/java/com/example/life_is_well/BMICalc.kt (hunk 1 above)
Comment (DerekKensinger) [reply app/src/main/java/com/example/life_is_well/BMICalc.kt:28]: @jacobburgo Will do moving forward.

File: app/src/main/java/com/example/life_is_well/BMICalc.kt (hunk 2 above)
Comment (DerekKensinger) [reply app/src/main/java/com/example/life_is_well/BMICalc.kt:20]: @jacobburgo that is exactly my thought.  Much better data management.

File: app/src/main/res/layout/activity_bmicalc.xml (hunk 3 above)
Comment (DerekKensinger) [reply app/src/main/res/layout/activity_bmicalc.xml:17]: It is ignoring one missing constraint on the Main Physical page that threw my UI all out of wack  It is was a suggestion that Android Studio gave me to fix the issue on that page.  I shouldn't be using it from here on out.  

When I use Linear Layouts, there are no constraints so if you see that in any of my linear layouts that is as intended.

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (DerekKensinger) [reply app/src/main/res/values/strings.xml:101]: @jacobburgo I will go ahead and add all my strings to the night/strings.xml file for my next Pull Request.

**Review by jacobburgo — APPROVED** [review]
Sweet. Good shit.

**Pull request merged**